doc/releases/changelog-dev.md

```diff
@@ -21,6 +21,9 @@
 * Fix Pytree serialization of operators with empty shot vectors:
   [(#6155)](https://github.com/PennyLaneAI/pennylane/pull/6155)
 
+* Fix PrepSelPrep template to work with torch:
+  [(#6191)](https://github.com/PennyLaneAI/pennylane/pull/6191)
+
 <h3>Contributors ✍️</h3>
 
 This release contains contributions from (in alphabetical order):
```

Comment:
Please add your name to the contribution list. (alphabetical order of the last name)